### DISCUSSION TITLE
Discuss the feasibility of using RUNWASI to reduce the size of Redash's Docker image.

### DISCUSSION THREAD

**Discussion by Jengro777** [RedashCommunity/redash#12] [General]
The current Redash Docker image size is 1.2 GB. Would building Redash as a RUNWASI Docker image significantly reduce the size of the Docker image?

1、wasmtime
2、wasmedge
3、wasmer

**Comment by justinclift** [comment]
When you say "WASI Docker image" that's kind of unclear to me.

My understand of WASI is that it's a webassembly interface standard, unless you're meaning a different thing?

If we want to reduce the size of the Docker image, we'd probably need to analyse what's taking up the most space first.

If it's the base OS taking up the most space, then we could move to something like an Alpine base image.  That's the the common approach. :smile:

If however the space is taken up by other things, we'd need to investigate those.
